Centaurus.DAL/Models/EffectsModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MongoDB.Bson;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.DAL.Models
+{
+    public class EffectsModel
+    {
+        public BsonObjectId Id { get; set; }
+
+        public long Apex { get; set; }
+
+        public int Account { get; set; }
+
+        public List<SingleEffectModel> Effects { get; set; }
+
+        public override string ToString()
+        {
+            return $"{{Effect apex: {Apex}, account: {(Account == default ? "system" : Account.ToString())} }}";
+        }
+    }
+
+    public class SingleEffectModel
```

Comment:
What's the point of adding `ApexIndex`? The order of the effect can always be retrieved from the container list. You'll always fetch the entire `EffectModel` from db, so you don't really need the index. Elements in the list will always retain the original order.
So maybe just `public List<byte[]> Effects { get; set; }` will be better in this case?

Reply:
During quantum processing, effects are grouping by account. `ApexIndex` contains effect's "absolute" index in the context of the current quantum.